tests/data/mock_manifests/InvalidFilenameAndEntityIDManifest.csv

```diff
@@ -2,5 +2,5 @@ Component,Filename,entityId
 MockFilename,schematic - main/MockFilenameComponent/txt1.txt,syn61682653
 MockFilename,schematic - main/MockFilenameComponent/txt2.txt,syn61682660
 MockFilename,schematic - main/MockFilenameComponent/txt3.txt,syn61682662
-MockFilename,schematic - main/MockFilenameComponent/txt4.txt,syn6168265
-MockFilename,schematic - main/MockFilenameComponent/txt5.txt,
+MockFilename,schematic - main/MockFilenameComponent/txt4.txt,syn1234567
```

Comment:
One note here, there may have been a reason for a manifest to have a empty synapse id as well. Should confirm with FAIR